app/helpers/bulk_updates_helper.rb

```diff
@@ -216,7 +226,7 @@ def display_value_title(update_type, value_type)
     "<h4>#{title}</h4><p>#{description}</p>".html_safe
   end
 
-  def science_keyword_for_display(keyword)
+  def science_keyword_for_display(type, keyword)
```

Comment:
maybe this should be renamed since it's being used for science and location keywords